pmpro-mailchimp.php

```diff
@@ -100,7 +100,18 @@ function pmpromc_sync_merge_fields_ajax()
 {
     //setup vars
     global $wpdb;
-    $api = pmpromc_getAPI();
+    // $api = pmpromc_getAPI();
+    $api = apply_filters('get_mailchimpapi_class_instance', null);
+    
+    if ( empty( $api ) ) {
+
+	global $pmpro_msg, $pmpro_msgt;
+	$pmpro_msg = __("Error loading MailChimp API", "pmpromc");

```

Comment:
Hi there, representing hiwebservices from the support forum. I don't think it's appropriate to always set `$pmpro_msg` here. I don't necessarily want the user to see an error if their signup succeeded and the only failure was MailChimp. I would appreciate a way to filter out this error reporting.